resources/server/players/player.service.ts

```diff
@@ -57,6 +57,14 @@ class _PlayerService {
     return this.getPlayer(source).getIdentifier();
   }
 
+  /**
+   * Returns the player phone number for a given source
+   * Will return null if no player is found online with that source
+   **/
+  getPhoneNumber(source: number): string {
```

Comment:
Do we really need this helper function? We already get it from the getPlayer metohd, and its feels like another layer of abstraction.

Reply:
Nah it’s not, will fix